README.md

```diff
@@ -139,6 +139,50 @@ Controllers that was tested on different host modes.
 
 ---
 
+### Building
+
+_The instructions below have been successful on Windows (11) but should mostly apply to Linux_
+
+1. (windows) Install Docker Desktop
+2. Setup pico sdk environment by following https://community.element14.com/products/raspberry-pi/b/blog/posts/working-with-pi-pico-c-c-projects-easily-a-docker-approach
```

Comment:
Fix the link style and put it on like this:  ... by following [this tutorial](https://community.element14.com/products/raspberry-pi/b/blog/posts/working-with-pi-pico-c-c-projects-easily-a-docker-approach)